ui/apps/platform/src/Containers/ComplianceEnhanced/Schedules/components/ReportJobs.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+function ReportJobs() {
+    return <div />;
```

Comment:
Remind me, will this component make the route-specific request for **compliance reports** jobs, and render the same reusable element for the report jobs table as for **vulnerability reports** jobs?

Reply:
Yeah, that is the plan. Ideally, we want to abstract out the rendering logic (as best we can) and reuse it in both places. I'll have to see how much can be refactored.